mojaloop-simulator/values.yaml

```diff
@@ -0,0 +1,430 @@
+# Default values for mojaloop-simulator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Usage:
+# Add simulators to the simulators object. The following example will create two simulators,
+# 'payerfsp' and 'payeefsp' that will be created with the default values available lower in this
+# file.
+#
+# simulators:
+#   payerfsp: {}
+#   payeefsp: {}
+#
+# The default values can be overridden for all sims by modifying mojaloop-simulator.defaults in
+# your parent chart. They can also be overriden per-simulator. The following example will result in
+# a payerfsp without a cache and a payeefsp with a cache.
+#
+# simulators:
+#   payerfsp:
+#     config:
+#       cache:
+#         enabled: false
+#   payeefsp: {}
+
+# TODO & notes:
```

Comment:
Is this todo and notes section needed?

Reply:
probably server as a good reminder for future tasks...you concerned that this being public, y'ld rather prefer them as stories?

Reply:
ok.. yeah probably.. I don't think this place has much visibility.. But I'll resolve this for now.